modules/compute/src/main/java/org/apache/ignite/internal/compute/loader/JobContextManager.java

```diff
@@ -100,8 +98,8 @@ public CompletableFuture<JobContext> acquireClassLoader(List<DeploymentUnit> uni
     }
 
     /**
-     * Creates a class loader for the given deployment units. The units will be acquired.
-     * The class loader will be closed when it is not used by any other job.
+     * Creates a class loader for the given deployment units. The units will be acquired. The class loader will be closed when it is not
+     * used by any other job.
```

Comment:
Reformatting unrelated javadocs for no good reason, aren't you?

Reply:
This is IDEA autoformatting